src/configuration.js

```diff
@@ -129,11 +133,30 @@ export class Configuration {
       return this.rules;
     }
 
-    this.rules = this.getRulesFromPaths(this.rulePaths);
+    const packageRules = this.getRulesFromPackages(this.rulePackages);
+    const pathRules = this.getRulesFromPaths(this.rulePaths);
+
+    this.rules = packageRules.concat(pathRules);
 
     return this.rules;
   }
 
+  getRulesFromPackages(rulePackages) {
+    const rules = new Set([]);
+
+    rulePackages.map(rulePackage => {
+      // We can't simply call `require()` because it needs to be from the project's node_modules
+      const rulePackagePath = ModuleResolver.resolve(
+        rulePackage,
+        path.join(process.cwd(), '__placeholder__.js')
+      );
```

Comment:
Ohhhh, that's interesting. I hadn't even considered that. 🤔 

Btw did the path you are passing _need_ to be a file and not a folder? In other words, the `__placeholder__.js` was required and is okay if it does not actually exist?

Reply:
@cjoudrey I actually borrowed this bit of code from the eslint packages: https://github.com/eslint/eslint/blob/a47770706ac59633dcd73e886d1a7282b324ee06/lib/init/config-initializer.js#L350

I actually don't know if a folder would work, but they probably added the placeholder for a reason? 😄If I remember when I'm finishing this PR, I'll see if it works with just the CWD.